### ISSUE TITLE
Accessibility issue with ScrollablePane and Sticky components

### ISSUE THREAD

**Issue by kejatura** [microsoft/fluentui#13974]
### Environment Information

- **Package version(s)**: 7.117.0
- **Browser and OS versions**: Chrome

### Describe the issue:

We currently use ScrollablePane  and Sticky components in conjunction with DetailsList in our project, very similar to this [example](https://developer.microsoft.com/en-us/fluentui#/controls/web/scrollablepane). When running the Accessibility Insights - Fast Pass, we have this issue : "Ensures aria-hidden elements do not contain focusable elements"

![image](https://user-images.githubusercontent.com/58444231/87092670-b61eb900-c1f0-11ea-8cbb-b3f90778a207.png)

This is also seen in **DetailsList Locked Header** example from : https://developer.microsoft.com/en-us/fluentui#/controls/web/scrollablepane

### Please provide a reproduction of the issue in a codepen:

codepen: https://codepen.io/kejatura/pen/zYrjVRd?editors=0010

1. Scroll down the detailslist (doesn't need to be scrolled to the bottom)
2. Launch Fast-Pass from Accessibility Insights for Web extension

#### Actual behavior:
Notice the error : "Ensures aria-hidden elements do not contain focusable elements"

#### Expected behavior:
Fixes mentioned:  "Focusable content should have tabindex='-1' or be removed from the DOM"

### Documentation describing expected behavior

Fixes mentioned:  "Focusable content should have tabindex='-1' or be removed from the DOM"

**Comment by paulgildea** [comment]
I wasn't able to a repo within codepen - this one might be able to repro the issue in a different form.

**Comment by xugao** [comment]
@kejatura - I can't seem to repro either. could you point out explicitly which element is set with `aria-hidden: true` and which element is focusable within?

**Comment by kejatura** [comment]
Hi @xugao, if you scroll down the detailslist (enabling the sticky header while scrolling) and run Fast Pass, it should have this error. The element that has the issue when I run the automated checks is:

<div aria-hidden="true" class="stickyAbove-48" style="height: 97px; left: 0px; right: 17px; top: 0px;">

**Comment by xugao** [comment]
@kejatura - thanks! i got the repro. Will look into fixing

**Comment by ayuspark** [comment]
Hi @xugao , our product in Dynamics also experiences the same bug. I wonder if you have an ETA for the fix or a timeline?
Thanks!
